lib/html_tag.js

```diff
@@ -11,7 +22,8 @@ function htmlTag(tag, attrs, text, escape = true) {
   for (const i in attrs) {
     if (attrs[i] === null || typeof attrs[i] === 'undefined') result += '';
     else {
-      if (i === 'href' || i === 'src') result += ` ${i}="${encodeURL(attrs[i])}"`;
+      if (i.endsWith('href') || i.endsWith('src') || i.endsWith('url')) result += ` ${escapeHTML(i)}="${encodeURL(attrs[i])}"`;
```

Comment:
Here are some attributes that might contain URLs:

- [`cite` attribute in `<blockquote>` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote)
- [`download` attribute in `<a>` tags](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a)
- [`style` attribute in any elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/style) might have `url()` inside.

Reply:
`<style>` can a bit tricky because double-quote and slash are commonly used. what about skip escapeHTML on it, while still encode `url()`?

Reply:
@curbengh I think that will be fine.

Reply:
@curbengh Also, some css property like `background` `background-image` could contains URL. It is now a little more tricky. I suggest just not to encode `<style>` tag and put a notice at docs.

Reply:
Addressed in latest commits.